sdk/extensions/Azure.AspNetCore.DataProtection.Blobs/README.md

```diff
@@ -0,0 +1,91 @@
+# Azure Storage Blob Key Store for Microsoft.AspNetCore.DataProtection
+
+The `Azure.AspNetCore.DataProtection.Blobs` package allows storing ASP.NET Core DataProtection keys in Azure Blob Storage. Keys can be shared across several instances of a web app. Apps can share authentication cookies or CSRF protection across multiple servers.
+
+## Getting started
+
+### Install the package
+
+Install the package with [NuGet][nuget]:
+
+```Powershell
+dotnet add package Azure.AspNetCore.DataProtection.Blobs -v 1.0.0-preview.1
+```
+
+### Prerequisites
+
+You need an [Azure subscription][azure_sub],
+[Storage Account][storage_account_docs] and [Storage Container][storage_container_docs] to use this package.
+
+To create a new Storage Account, you can use the [Azure Portal][storage_account_create_portal],
+[Azure PowerShell][storage_account_create_ps], or the [Azure CLI][storage_account_create_cli].
+Here's an example using the Azure CLI:
+
+```Powershell
+az storage account create --name MyStorageAccount --resource-group MyResourceGroup --location westus --sku Standard_LRS
+az storage container create --account-name MyStorageAccount -n mycontainer
+```
+
+## Examples
+
+To enable persisting keys to Azure Blob Storage call the `PersistKeysToAzureBlobStorage` method. The `Uri` provided has to be a blob URI in the following form `https://MyStorageAccount.blob.core.windows.net/mycontainer/myblob`.
```

Comment:
Nit - we've been doing `https://{storage_account}.blob.core.windows.net/{container}/{blob}` when we format these elsewhere.